### ISSUE TITLE
 Argument of type 'FloatArray' is not assignable to parameter of type 'Point[]'

### ISSUE THREAD

**Issue by grahambates-sp** [kylebarron/snap-to-tin#32]
When I clone the repo and try to build it, I get the following error.

````src/index.ts:118:9 - error TS2345: Argument of type 'FloatArray' is not assignable to parameter of type 'Point[]'.
  Type 'Float32Array' is missing the following properties from type 'Point[]': pop, push, concat, shift, and 5 more.

118         handleLineString(lineSegment, this.index, this.triangles)
            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

````

**Comment by kylebarron** [comment]
I’ve been developing on master, so as you can tell it’s not stable. 0.1.0 was published to npm as @kylebarron/snap-features-to-tin, but then I decided I wanted a shorter name, and it’s not yet published with the shorter name.

I’ll ping you in a day or two once I’ve sorted things out a little more. Alternatively you can comment out `snapLines` and `snapPoints` in index.ts , if you want to test out `snapFeatures`

**Comment by kylebarron** [comment]
Ok this should be fixed now and you should have no problems building master. I haven't fully built out tests yet, so if you try it and have an issue let me know

**Comment by kylebarron** [comment]
I also published 0.2.0 to npm
